src/Component/Movies.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Component } from "react";
+import '../App.css'
+
+
+class Movies extends Component{
+    constructor() {
+        super()
+        this.state = {
+            movies: []
+        }
+    }
+fetchFilms = () => {
+    fetch("https://ghibliapi.herokuapp.com/films")
+        .then((res) => res.json())
+        .then((data)=>{
+            this.setState({
+                movies: data,
+            })
+        });
+};
+    render () {
+        let showMovieArr = this.state.movies.map((movies)=>{
+            return (
+                <div className="movies-card">
+                    <ul>
+                       <li>Name: {movies.title}</li>
+                       <li>Release Date: {movies.releasedate}</li>
+                       <li>Description: {movies.description} </li>
+                    </ul>
+                </div>
+            )
```

Comment:
```suggestion
            return (
                <div className="movies-card">
                    <ul>
                       <li>Name: {movies.title}</li>
                       <li>Release Date: {movies.releasedate}</li>
                       <li>Description: {movies.description} </li>
                    </ul>
                </div>
            )
```
You don't want an entire array of movie cards. On this page you're only supposed to display **one movie at a time**. You do, however, want to display **_all_ the movie titles in a dropdown menu**, and map IS a useful tool for doing that. How would you change this to display every movie title in a dropdown?